packages/widgets-internal/pool/Apr.tsx

```diff
@@ -101,29 +125,100 @@ export function Apr<T>({
 
   const isValidate = apr !== undefined && !Number.isNaN(apr);
 
+  const tooltipStakeApy = useMemo(() => {
+    const currentApr = vaultKey ? rawApr : apr;
+    return `${currentApr?.toLocaleString("en-US", { maximumFractionDigits: 2 })}%` ?? "0%";
+  }, [vaultKey, rawApr, apr]);
+
+  const boostedAprGreaterThanZero = useMemo(() => new BigNumber(boostedApr ?? 0).isGreaterThan(0), [boostedApr]);
+
+  const { targetRef, tooltip, tooltipVisible } = useTooltip(
+    <Box>
+      <Box>
+        <Box>
+          <Text bold as="span">
+            {t("Total APY:")}
+          </Text>
+          <Text bold as="span" ml="4px">
+            {`${poolApr?.toLocaleString("en-US", { maximumFractionDigits: 2 })}%`}
+          </Text>
+        </Box>
+        <Box>
+          <Text bold as="span">
+            {t("Fee APY:")}
+          </Text>
+          <Text bold as="span" ml="4px">
+            {`${boostedApr?.toLocaleString("en-US", { maximumFractionDigits: 2 })}%`}
+          </Text>
+        </Box>
+        <Box>
+          <Text bold as="span">
+            {t("Stake APY:")}
+          </Text>
+          <Text bold as="span" ml="4px">
+            {tooltipStakeApy}
+          </Text>
+        </Box>
+      </Box>
+      <Text mt="10px" lineHeight="120%">
+        {t(
```

Comment:
I think we should have different tooltip for different type of booster. For example if next time we have ALP and other booster provider, then we may have two booster aprs and need to tell which apr belongs to which provider